{"flow_id": "DataFlow", "run_number": "1713619301004251", "step_name": "prepare_dataset", "task_id": "2", "field_name": "attempt_ok", "type": "internal_attempt_status", "value": "True", "tags": ["attempt_id:0"], "user_name": "amrutagokhale", "ts_epoch": 1713619314786}